src/newSeed/stage3.ts

```diff
@@ -49,6 +50,9 @@ export class Stage3 extends BaseStage {
       message = "Please enter a valid address for the Seed Token Address";
     } else if (!this.seedConfig.tokenDetails.maxSeedSupply || this.seedConfig.tokenDetails.maxSeedSupply === "0") {
       message = "Please enter a non-zero number for Maximum Supply";
+    } else if (this.seedConfig.seedDetails.fundingMax && this.seedConfig.seedDetails.pricePerToken &&
+      Number(fromWei(this.seedConfig.seedDetails.fundingMax)) > Number(fromWei(this.seedConfig.tokenDetails.maxSeedSupply)) * Number(fromWei(this.seedConfig.seedDetails.pricePerToken))) {
```

Comment:
Our practice everywhere is to convert the bignumber to a string and pass that to `numberService.fromString`.  Can you please follow that here?  Thanks.